### PR TITLE
feat: sync connect status

### PR DESCRIPTION
**Pull request by Keith-CY**

### CODE REVIEW

File: packages/neuron-wallet/src/utils/initApp.ts [hunk 1]
```diff
@@ -15,8 +18,25 @@ const initiateNetworks = () => {
...
+    () => {},
+    () => {
+      // TODO: handle complete
+    },
+  )
+}
+
 const initApp = () => {
   nodeService.start()
+  // TODO: this function should be moved to somewhere syncing data
```
Comment (ashchan) [inline packages/neuron-wallet/src/utils/initApp.ts]: Before we move this, logically checking should happen after init.

Comment (Keith-CY) [reply packages/neuron-wallet/src/utils/initApp.ts]: I'm not sure what does the checking mean, if it refers to error catch, which is handled in [service](https://github.com/nervosnetwork/neuron/pull/244/files#diff-33a5ced4a8dfd71e03e7e1352a3ebbfeR27)

Comment (ashchan) [reply packages/neuron-wallet/src/utils/initApp.ts]: Without looking deep into `checkConnectStatus` and `initConnection`, these two methods seems to be related to each other by reading their names. So it's very strange for me to understand why `check` happens before `init`.

Comment (Keith-CY) [reply packages/neuron-wallet/src/utils/initApp.ts]: > Without looking deep into `checkConnectStatus` and `initConnection`, these two methods seems to be related to each other by reading their names. So it's very strange for me to understand why `check` happens before `init`.

got the point, will it be better to rename the method to `syncConnectStatus` ?

Comment (ashchan) [reply packages/neuron-wallet/src/utils/initApp.ts]: That'll look good.

Comment (Keith-CY) [reply packages/neuron-wallet/src/utils/initApp.ts]: updated in https://github.com/nervosnetwork/neuron/pull/244/commits/5d877b5678ac58547f2a1e59a3de1cb5794573c0

**Pull request merged**